asyncy/Exceptions.py

```diff
@@ -120,6 +120,40 @@ def __init__(self, story=None, line=None, service=None, action=None):
             story=story, line=line)
 
 
+class OmgPropertyValueMismatchError(AsyncyError):
+    """
+    Raised when value does not match type.
+    """
+    def __init__(self, value, type):
+        super().__init__(
+            f'The property value {value}'
+            f'should be of type {type}.'
+            f'It does not adhere to the microservice '
+            f'service contract.')
+        self.value = value
+        self.type = type
+
+    def __str__(self):
+        return(str(self.value))
```

Comment:
```suggestion
        return repr(self.value)
```

Also, as mentioned in earlier review we shouldn't even overwrite `__str__` here. What's the reason?